app/controllers/metrics_controller.rb

```diff
@@ -12,9 +14,11 @@ def index
                                    else
                                      course_max[1]
                                    end
+
+    @allow_ca = course.watchlist_configuration.allow_ca
```

Comment:
Same comment about the safe navigation operator (see the comment on `assessments/index.html.erb`)

![Screenshot 2022-11-25 at 02 12 59](https://user-images.githubusercontent.com/9074856/203922534-8720a4f8-7058-42d8-b682-a1dc35fffaef.png)